automation_tools/__init__.py

```diff
@@ -3245,3 +3245,33 @@ def setup_rhv_ca():
         '/etc/pki/ca-trust/source/anchors/'.format(http_server))
     run('update-ca-trust enable ; update-ca-trust')
     print("RHV CA cert has been successfully added to CA trust")
+
+
+def configure_telemetry():
+    """Setup telemetry on the satellite box with grafana monitoring tool
+    """
+    http_server = os.environ.get('HTTP_SERVER_HOSTNAME')
+    print("Install required packages for PCP")
+    run('yum -y install pcp pcp-pmda-apache')
```

Comment:
do we need any repo enabled for those packages?

Reply:
no